src/Authentication/Helpers/AuthenticationHelper.cs

```diff
@@ -20,6 +20,11 @@ namespace Altinn.Platform.Authentication.Helpers
     /// </summary>
     public static class AuthenticationHelper
     {
+        /// <summary>
+        /// Regular expression for valid uri
+        /// </summary>
+        public static readonly Regex ValidUri = new Regex(@"^https://[a-zA-Z0-9.-]+.[a-zA-Z]{2,}$");
```

Comment:
This isn't used? And if it is, should be compiled.

Reply:
removed